js/Blog.json

```diff
@@ -0,0 +1,31 @@
+{
+  "posts": [
+    {
+      "title": "Hello World",
+      "body": "Lorem ipsum dolor sit amet consectetur adipisicing elit. Commodi, quibusdam. Officiis veritatis quis praesentium natus distinctio molestias magni reprehenderit sapiente!",
+      "author": "John Doe",
+      "published_on": "20 November 2019",
+      "source": "medium",
+      "link": "https://medium.com",
+      "image": "../img/zealicon.png"
```

Comment:
For now, this is fine but later we will have hosted image links here

Reply:
Also, don't save the blog website image in the images folder we can also add their link here.

Reply:
yes sir, links will also work as [image] value is being rendered for background URL.